src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Wrapper, List, NavLink } from './styled'
+
+export const Navbar: React.FC = () => {
+	return (
+		<Wrapper>
+			<List>
+				<li>
+					<NavLink to="/my-profile/timeline">Timeline</NavLink>
```

Comment:
What is timeline?

Reply:
I didn't know what to write there, so I wrote what I could think of. I thought it was better than: Button1, button, 2, button3...

Reply:
Got it, but I don't think we need profile to have multiple pages under itself when we don't even know what they would be. :sweat_smile:

Reply:
ok